Chat/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "./common.tsconfig.json",
```

Comment:
nothing else extends the common tsconfig - consider merging in and just having one tsconfig file

Reply:
It will be easier to migrate in changes from the UI Library so lets keep this in for now.